pkg/monitor/monitorapi/types.go

```diff
@@ -60,20 +60,135 @@ func EventLevelFromString(s string) (EventLevel, error) {
 }
 
 type Condition struct {
```

Comment:
At some future date I think we'll want `RelatedObjects []Locator` to allow linkage of some kind.